packages/test/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "scripts": {
     "pretest": "npm run asbuild",
-    "asbuild": "asc assembly/index.ts --target debug",
+    "asbuild": "asb --debug",
```

Comment:
Why can't this just be without another test dependency since the test doesn't even have a debug entry in the asconfig?

Please remove it.